lib/main.dart

```diff
@@ -17,10 +21,17 @@ class LachiesLifePlannerApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      debugShowCheckedModeBanner: false,
-      title: 'Lachie\'s Life Planner',
-      home: HomeScreen(),
+    return MultiBlocProvider(
+      providers: [
+        BlocProvider<TasksBloc>(
+          create: (context) => TasksBloc(tasksRepository: TasksRepository())..add(LoadTasksEvent()),
+        )
```

Comment:
This multi-provider will get bigger overtime.  We should extract this to another widget and call it `AllProviders`